### PR TITLE
testing_user_controller_and_user_course and sign in page: 1st time

### PR DESCRIPTION
**Pull request by mdmahmudurrahman**

### CODE REVIEW

File: app/controllers/supervisor/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Supervisor::UsersController < ApplicationController
   before_action :load_user, only: [:show, :edit, :destroy]
-  authorize_resource
+    authorize_resource
```
Comment (noor-framgia) [inline app/controllers/supervisor/users_controller.rb]: remove space

File: db/schema.rb [hunk 2]
```diff
@@ -92,9 +92,9 @@
     t.integer  "user_id",      limit: 4
     t.integer  "course_id",    limit: 4
     t.datetime "deadline"
-    t.boolean  "is_completed",           default: false
-    t.datetime "created_at",                             null: false
-    t.datetime "updated_at",                             null: false
+    t.boolean  "is_completed"
```
Comment (noor-framgia) [inline db/schema.rb]: default: false

File: spec/factories/user.rb [hunk 3]
```diff
@@ -3,8 +3,12 @@
...
     name {Faker::Name}
-    role 0
     email {Faker::Internet.email}
     password {Faker::Internet.password 6}
+    role 1
+    factory :supervisor do
+      role 0
+    end
   end
+
```
Comment (noor-framgia) [inline spec/factories/user.rb]: remove

**Comment by manazirahsan** [comment]
Okey

**Comment by noor-framgia** [comment]
ok

**Pull request merged**